Tests/create_mock_user.py

```diff
@@ -85,6 +84,19 @@ def add_some_photos(token):
             filename = random.choice(files)
             file_path = os.path.join(directory, folder, filename)
 
+            if type == "Upper garment":
```

Comment:
Placeholder z kilkoma podstawowymi typami na potrzeby generowania zestawów

Reply:
Czy nie lepiej dodawać "prawdziwe" tagi pochodzące z nazwy folderów? wyżej można zrobić
        if folder in ["dress", "jumpsuit", "outwear", "top"]:
            type = "Upper garment"
            tags=folder